.github/dependabot.yml

```diff
@@ -88,3 +88,8 @@ updates:
     ignore:
       - dependency-name: "*"
         update-types: ["version-update:semver-patch"]
+
+  - package-ecosystem: "cargo"
+    directory: "/pallets/offchain-task-scheduler"
+    schedule:
+      interval: "weekly"
```

Comment:
Can you move this up next to the other cargo sections for better visibility. Order alphabetically if possible.